lib/elixir_internal_certificate/accounts/accounts.ex

```diff
@@ -0,0 +1,122 @@
+defmodule ElixirInternalCertificate.Accounts do
```

Comment:
With out convention, this module should be `ElixirInternalCertificate.Account.Accounts`